### PR TITLE
[18.0][MIG] storage_backend: Migration to 18.0

### PR DESCRIPTION
**Pull request by thienvh332**
OCA Port:
- https://github.com/OCA/storage/pull/406

### CODE REVIEW

File: storage_backend/README.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
+==============
+Storage Bakend
```
Comment (henrybackman) [inline storage_backend/README.rst]:
```suggestion
Storage Backend
```

File: storage_backend/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Storage Bakend",
```
Comment (henrybackman) [inline storage_backend/__manifest__.py]:
```suggestion
    "name": "Storage Backend",
```

**Review by henrybackman — APPROVED** [review]

**Review by mmequignon — APPROVED** [review]

**Comment by lmignon** [comment]
@thienvh332 IMO you should migrate to `fs_storage`

**Pull request merged**